instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/BaseSessionReplayTest.kt

```diff
@@ -13,6 +13,8 @@ import androidx.test.platform.app.InstrumentationRegistry
 import com.datadog.android.Datadog
 import com.datadog.android.rum.GlobalRumMonitor
 import com.datadog.android.sdk.integration.RuntimeConfig
+import com.datadog.android.sdk.integration.sessionreplay.BaseSessionReplayTest.MatchingStrategy.CONTAINS
+import com.datadog.android.sdk.integration.sessionreplay.BaseSessionReplayTest.MatchingStrategy.EXACT
```

Comment:
are these unused imports?

Reply:
we were using ` MatchingStrategy.CONTAINS` and ` MatchingStrategy. EXACT` in the code before, the imports are automatically added by IDE reformat, I remove the qualifier name in the code now.